Set/Set/Card.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  Card.swift
+//  Set
+//
+//  Created by Yeliena Khaletska on 12.06.2024.
+//
+
+import Foundation
+import UIKit
+
+struct Card {
+    let color: Card.Color
+    let shade: Card.Shade
+    let shape: Card.Shape
+    let number: Card.Number
```

Comment:
Since we are inside of `Card` context it is fine to omit this. But, if you feel you benefit from them then surely leave it as it is right now
```suggestion
    let color: Color
    let shade: Shade
    let shape: Shape
    let number: Number
```